foyer-memory/src/cache.rs

```diff
@@ -339,6 +339,12 @@ where
     V: Value,
     S: HashBuilder,
 {
+    /// Set the capacity of the in-memory cache instance.
+    pub fn with_capacity(mut self, capacity: usize) -> Self {
+        self.capacity = capacity;
+        self
+    }
```

Comment:
Btw, it is a little bit verbose to provide another function to set capacity since it is required when creating. Would you mind to remove it? 🙏